bluesky/tests/conftest.py

```diff
@@ -10,6 +10,15 @@ def fresh_RE(request):
     loop.set_debug(True)
     RE = RunEngine({}, loop=loop)
     RE.ignore_callback_exceptions = False
+
+    def clean_al():
```

Comment:
Is this supposed say `clean_all`? Maybe `al` stands for something that I'm not seeing; if so, I'll leave this :sheep: here preemptively.